quran/quran.com-frontend-next
# Add advanced verse copy feature

### Summary
This PR adds advanced verse copy feature.

### Test Plan
1. Move to a chapter or a verses's page e.g. https://quran-com-frontend-next-zeta.vercel.app/1/1.
2. Click on the verse's actions icon to the left side of the verse.
3. A drop-down should appear that contains `Advanced Copy` item.
4. Click on `Advanced Copy` and a modal should appear that contains all the options for advanced copy.

### Screenshots

| Before | After |
| ------ | ------ |
| <img width="167" alt="Screen Shot 2021-08-02 at 16 39 58" src="https://user-images.githubusercontent.com/15169499/127840642-7fff6acd-3370-4ed5-a27b-ce99c8602605.png"> | <img width="247" alt="Screen Shot 2021-08-02 at 16 29 25" src="https://user-images.githubusercontent.com/15169499/127840736-7724e333-caff-40d4-a573-5d76c996678b.png"> |
|  | <img width="750" alt="Screen Shot 2021-08-02 at 16 29 06" src="https://user-images.githubusercontent.com/15169499/127840842-629b4a58-d9c4-442c-883c-ede204cbbc29.png"> |
|  |<img width="754" alt="Screen Shot 2021-08-02 at 16 29 17" src="https://user-images.githubusercontent.com/15169499/127840856-51eb7511-dbe9-4eda-b1ca-60d19eed1bec.png"> |
